src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -40,7 +40,7 @@ class CromwellApiServiceActor(val workflowManager: ActorRef, swaggerService: Swa
   implicit def executionContext = actorRefFactory.dispatcher
   def actorRefFactory = context
 
-  def possibleRoutes = options { complete(StatusCodes.OK) } ~ docsRoute ~ swaggerService.uiRoutes ~ workflowRoutes
+  def possibleRoutes = options { complete(StatusCodes.OK) } ~ docsRoute ~ swaggerService.swaggerUiRoutes ~ workflowRoutes
```

Comment:
For the current utils in lenthall, yes one needs the `docsRoute` too. The `docsPath` property at the moment isn't used for routing, only as a relative browser path where the `index.html`, inside the `swagger-ui` jar file, can load the documentation yaml or json. In this case, the relative path is `swagger/thurloe.yaml` that ends up getting resolved to `http://server/swagger/thurloe.yaml`.

I didn't want to assume someone wouldn't still be using `spray-swagger` instead of static yaml, & didn't make the lenthall utility auto search for the `docsPath` within the classpath resources. I guess I was overly paranoid of accidentally exposing the wrong files auto-magically through routes. I believe the spray-spec also says that the top level yaml can also point to other child files, but we can cross that bridge some other time.